webapp/models.py

```diff
@@ -50,9 +51,12 @@ class CVE(db.Model):
     __tablename__ = "cve"
 
     id = Column(String, primary_key=True)
-    published = Column(DateTime)
+    published = Column(DateTime(timezone=True))
```

Comment:
I know we discussed this when pairing and decided it was okay. I'm just wondering, given the recent discussion we had about API versioning with regard to Microsoft etc., do you think this might break anyone? Might anyone be relying on `cve.published` *not* containing a timezone?

Reply:
I'm not sure really, but since this isn't really a necessary change maybe it's best to err on the side of caution and leave it without the timezone?